docs/docs/rest-api/public/api/v2/apps.raml

```diff
@@ -0,0 +1,451 @@
+
+###
+# Get a list of applications
+#
+get:
+  description: Get the list of running applications. Several filters can be applied to filter the result list.
```

Comment:
Should we mention that the call is/might get expensive? Is there a raml-way of adding a note/mark other than mis-using traits for that?